iOS Client/PodcastsController/PodcastsCoordinator.swift

```diff
@@ -43,10 +48,45 @@ class PodcastsCoordinator: Coordinator, StoreSubscriber {
     
     // MARK: - StoreSubscriber
     func newState(state: MBAppState) {
+        
+        switch state.podcastsState.player {
+        case .initialized:
+            break
+        case .playing:
+            if let podcast = state.podcastsState.selectedPodcast, podcast.guid != currentPlayingPodcast?.guid {
+                start(podcast)
+                currentPlayingPodcast = state.podcastsState.selectedPodcast
+            } else if case .paused = playerState {
```

Comment:
can we remove the condition and just do `else { player.play() }` ?

Reply:
I think it will just have no effect if it is already playing, and it will cover us if somehow the player is in an `error` state

Reply:
No so I tried that but the podcast will start over on player.play()